### ISSUE TITLE
Couldnot find stream object.

### ISSUE THREAD

**Issue by santhoshreddyk** [rtc-io/rtc-screenshare#10]
It seems the current version of screenshare module is entirely differnt from earlier version.
I used to attach captured stream to the quickConnect (qConnect.addStream(stream)) object using the below code.

shareScreen = function(){
	screenshare.window(function(err, constraints) {
	  if (err) {
		return console.error('Could not capture window: ', err);
	  }
	  console.log('attempting capture with constraints: ', constraints);
	  localMedia = media({ constraints: constraints })
		.on('error', function(err) {
		  console.error('Could not capture: ', err);
		});
		localMedia.once('capture', function(stream) {
		streamObj = stream;
			qConnect.addStream(stream);
		});
		localMedia.render(local);
	});
}

But the current api is not having the "once" function in it, and no similar method is there it seems. I want to send my captured stream to my friend through a quickconnect datachannel to have a realtime desktop sharing app. How do i do this with current api?

There is bundle.js file, how do i get that file when i am running the screen share application with quickconnect server. 
i could not understand the meening of sharewindow.js:bundle.js exactly in (beefy sharewindow.js:bundle.js command), who is generating bundle.js and where is it residing.
How do i get bundle.js if iam running quickconnect server instead of beefy?

**Comment by DamonOehlman** [comment]
Yes - there have been quite a few changes since you last worked with the code it seems.  I've added some more up-to-date instructions on using the screenshare module and how to prep things for deployment in the [demo-screenshare README](https://github.com/rtc-io/demo-screenshare)

**Comment by DamonOehlman** [comment]
@santhoshreddyk Just checking in to see how you went upgrading to the latest version.  Also the new [`rtc-screen`](https://github.com/rtc-io/rtc-screen) package might be of interest to you also.  Same stuff under the hood, just a higher level API which actually does to the capture for you instead of just returning constraints...
